src/code/code_800FCE80.c

```diff
@@ -132,6 +140,11 @@ f32 Math_FAtanF(f32 x) {
     }
 }
 
+/**
+ * @param y
+ * @param x
```

Comment:
feel like these param comments can just be omitted especially cause they arent adding any explanation or additional context. goes for most of these functions